Demos/GameLoop.cs

```diff
@@ -23,7 +23,17 @@ public GameLoop(Window window)
         {
             Window = window;
             Input = new Input(window, Pool);
-            Surface = new RenderSurface(window.Handle, window.Resolution, enableDeviceDebugLayer: false);
+            Surface = new RenderSurface(
+#if OPENGL
+                window.WindowInfo,
+#else
+                window.Handle,
+#endif
+                window.Resolution
+#if DEBUG
+                , enableDeviceDebugLayer: true
+#endif
```

Comment:
This will cause crashes on windows for people who don't have the necessary developer SDKs installed when they try to run in debug mode (as I found out :P). Should remain false.